examples/supply-chain-app/Dockerfile

```diff
@@ -0,0 +1,139 @@
+FROM node:12.20.1-alpine3.12 as builder
+
+RUN apk update
+
+# Need JDK for the openapi-generator that is part of the build process
+RUN apk add --no-cache openjdk8=8.252.09-r0
+
+# Need git because some of our npm depedencies might be coming
+# straight from github instead of being an npm package on npmjs.com.
+RUN apk add --no-cache git
+
+# Some install scripts of the npm package fabric-network need python
+RUN apk add --no-cache python3 py3-pip
+
+RUN npm install modclean -g
+
+WORKDIR /
+RUN mkdir /app/
+WORKDIR /app/
+COPY ./ ./
+RUN npm ci
+RUN ./node_modules/.bin/lerna clean --yes
```

Comment:
Why you did not use npx ?

Reply:
@sfuji822 Usually I leverage npx to stay operating system neutral with the commands I'm executing, but in the container there's no need for it because the container image is fixed for a specific OS that I know is guaranteed to work with the syntax of `./node_moules/.bin/...`.
The other part of it is that npx is slower than directly invoking the executable file so that's why I only use npx when I want to make sure that the command works on different operating systems as well (for example I had bugs some time ago where windows was failing with the forward slashes).

Reply:
I got it. It will be OK for now.

Reply:
@sfuji822 Cool, thanks. Any chance that you could approve the pull request? The governance says that it cannot be merged without approval.